Add flag to decompress brotli server-side (#2854)
ordinals/ord: docs/src/guides/testing.md

```
ord -r server
```

> If you want to test Brotli encoded content over http you need to add:

Reviewer: ```suggestion

By default, browsers don't support compression over HTTPS. To test compressed content over HTTP, add the `--decompress` flag:
```

Author: we're only decompressing brotli so made it explicit

```
ord -r server
```

> By default, browsers don't support compression over HTTPS. To test compressed

Reviewer: ```suggestion

By default, browsers don't support compression over HTTPS. To test compressed
```

> content over HTTP, add the `--decompress` flag:

Reviewer: ```suggestion
content over HTTP, use the `--decompress` flag:
```
